code/MUT/MUT/Options.cs

```diff
@@ -28,6 +30,11 @@ public string File
         {
             get { return (null == arguments["--file"]) ? null : arguments["--file"].ToString(); }
         }
+
+        public string Tag
+        {
+            get { return (null == arguments["--tag"]) ? null : arguments["--tag"].ToString(); }
```

Comment:
What about checking for arguments["-t"]? Although it looks like the file argument doesn't check for -f either.